### ISSUE TITLE
1.4.0 Store-Centric RC UI/UX Issues

### ISSUE THREAD

**Issue by dstrukt** [btcpayserver/btcpayserver#3180]
Wanted to maintain for visibility and prioritization in parallel for issues out of scope of (#3091) Will update as issues are addressed, this is still a bit of a mess as we work through the remaining bits of a large PR.

P1: essential
P2: ideal, make decision on
P3: nice to have, but could release without

### Sidebar:

- [x] P1: Remove global context switch
- [x] P1: Apps don't switch based on store selection
- [x] P1: Wallet Empty state before configuration (See Figma).
- [x] P1: The "payments > wallets" tab item view should replace the top "wallet" view after creation instead of a new tab (per Figma mocks), will lead to dedicated Wallet page (with settings, etc..)
- [x] P1: Lightning Node Landing View Update (See Figma .. might be only P1 issue we don't resolve)
- [x] P1: "Wallet" sidebar item should still be selected on all tab views (only works on "Settings" tab currently)
- [x] P1: "Invoices" sidebar has no active state
- [x] P1: "Account" sidebar has no active states, for any of the subpage items
-
- [x] P2: limit the number of characters in the Store Selector
- [x] Fixed sidebar width? (240 or 280px)
- [x] Add ellipses on long display App titles (see example below) (@dennisreimann)
- [x] P2: Combining Account/ThemeSwitch/Logout into a functional component to reduce # of items on left sidebar
- [x] Add store icon to the StoreSelector dropdown (@dennisreimann)
-
- [x] P3: make left/right padding changes on sidebar (as mentioned on PR by @dstrukt)
- [x] P3: Remove plugin "status" element
- [x] P3: Left align store copy (name) in StoreSelector dropdown element (@dennisreimann)
- [x] P3: Change margin-top above StoreSelector from -s to -m (@dstrukt)

### Content:
- [x] P1: Hide/Remove the deprecated Payment Methods > Wallet component (@dennisreimann)
- [x] P1: Hide/Remove the deprecated Payment Methods > Lightning component (@dennisreimann)
- [x]  P1: remove tabbed navigation for payouts, pay button, and pull payments
-
- [x] P2: "Create app" is a modal .. or a full page? Modal might be more fitting.
- [x] P2: Tabbed nav mobile & tablet improvement (@dennisreimann)
- [x] P2: clean up the payouts view (@dstrukt - in progress)
- [x] P2: Noticeable time delay in creating apps, feel like this has been an issue for a while if I remember
- [x] P2: Remove all instances of "Back to list" when the time comes @dennisreimann 
-
- [x] P3: "{Store Name}" - > "Store Settings" header .. ?
- [x] P3: "Settings" -> "Store Settings" sidebar label
- [x] P3: footer container should be the same width as the main content, the left/right flex push is fine @dennisreimann 
- [x] P3: .mb-5 -> .mb-4 class under header, above tab navigation (@dstrukt)
- [x] P3: Change main content section padding left & right from "l" to "xl", ideally 80px, but the "xl" 64px would be sufficient until we can address spacers in the design system follow up changes
- [x] P3: fix search filter break on tablet (@dstrukt)
- [x] P3: update and improve all icons (@dstrukt)
- [x] P3: consistent  main sub-page CTA paddings (@dstrukt)
- [x] P3: consistent  header sizings (@dstrukt)
- [x] P3: adjust "off" color state for the wallet/lightning, red to yellow for "pending", red only for error (@dennisreimann)

----

### General Thoughts:
- I'm still not completely sold on the switch, the context switch is jarring, and would be confusing to users. might want to do some exploration in figma, but maybe our next discussion will sway my view (Architectural)
- Continue to consolidate the store settings navigation, definitely feel we could cut 1-2 items (Content)
----

### 1.4.1
Tracking, but will remove.
- [x] P3: move search inputs on main views to left (@dstrukt)

----
### 1.5.0  Notes:
- P1: definitely worth attempting to combine "payment requests" and "invoice", and improve the general UX around creation/sharing .. have some ideas to move that forward for 1.5.0
- P2: improve some of the empty states views (invoices, pay. req., payouts, etc..) - @dstrukt
- P2: improve the hot wallet confirmation view (dstrukt), might have time to address for 1.4.0
[might update as I think of them, but will likely use this ticket as a tracking list to remove as we do]</div>

--- 
### Examples:
<img width="271" alt="Screen Shot 2021-12-20 at 1 36 20 PM" src="https://user-images.githubusercontent.com/6250771/146847347-f1ecff11-3e56-4012-b35e-1a8ee34e089d.png">

**Comment by dennisreimann** [comment]
Done in #3205 are:

- P1: Apps don't switch based on store selection
- P1: remove tabbed navigation for payouts, pay button, and pull payments
- P3: "{Store Name}" - > "Store Settings" header .. ? (Removed, it felt redundant)
- P3: "Settings" -> "Store Settings" sidebar label

**Comment by dstrukt** [comment]
P3: "{Store Name}" - > "Store Settings" header .. ? (Removed, it felt redundant)

It is a bit redundant, but it feels very empty without the title.
